test/integration/test.js

```diff
@@ -0,0 +1,800 @@
+const fs = require('fs');
+const chai = require('chai');
+const chaiHttp = require('chai-http');
+const logger = require('../../logger');
+const expect = chai.expect;
+
+if (!process.env.TEST_API_URL || !process.env.TEST_API_ACCESS_TOKEN) {
```

Comment:
How do these get set when travis runs?

Reply:
You can set them in [Travis CI](https://travis-ci.org/MozScout/scout-ua/settings). You have the option to encrypt the environment variables so they are not exposed